### ISSUE TITLE
Xact bulk update

### ISSUE THREAD

**Issue by ZS-Man** [FrSkyRC/ETHOS-Feedback-Community#2956]
Hi, after #989 is done and working, what about bulk update all connected(FBUS) servos at once? OTA or S.port from TY? Is or will be possible?

**Comment by bsongis-frsky** [comment]
Interesting feature!

**Comment by lpott6** [comment]
An expansion of this feature would be very helpful once the servos are installed into a model. Right now it seems that only the servo on Pin 1 can be configured using the XACT script. It is difficult to move each servo to Pin 1 in the model for configuration. It would be great to be able to "select the Pin to configure" and then configure the servo attached to that Pin.

I will add this as a new feature request if that would be better.
Thanks.

**Comment by ZS-Man** [comment]
Hi, are there any progress?

A mass update of all servos at once might not be ideal, if there are different types of Xact servos in the model. 

Possible idea:
1. When I select OTA update on the firmware file, ETHOS will allow me to select the desired channel (servo)

or
2. add the "FW update" option to the device config for each servo separately, and the option to select the desired firmware file.

Or can ETHOS itself detect the exact servo type (from telemetry? ) and automatically send the correct FW only to the correct type?

**Comment by ZS-Man** [comment]
Hi, after a few months I would like to remind you of this request. Is there a chance that it will be implemented in ETHOS?

**Comment by frskyrc-ben** [comment]
So my suggestion is a little bit different than the issue topic here, but one thing I would think can be very useful is an XACT ID and Channel setting wizard that will walk you through programming multiple servos with the basic settings of ID and Channel. 

For example, you can select the total # of servos to program, then it will give you each line with different servo Physical ID, Application ID and Channel assignments. You plug the servos 1-by-1 into the top of the radio and click Save to Flash to program those settings to the servo and progress through each servo after each setting is completed.

Here is a simple mockup of how I envision this might look 

![image](https://github.com/user-attachments/assets/a0345da5-01e1-4a24-9847-3d8d2585bc6b)

I think this would make the initial setup for the servos a lot easier and then the more advanced programming can be done through the Telemetry - Device Config to adjust center point, direction or other fine tuning as needed.

**Comment by patty-cakess** [comment]
As someone who will kit out an entire plane in FrSky servos, the setup would be 100% better if this was possible, talking about FrSky Ben's idea.

**Comment by ZS-Man** [comment]
Yes, this looks really nice and it will definitely make the initial setup of the servos easier.
But again I would like to remind you about the possibility of OTA FW update of different types of servos, without disconnecting from the model.... :-))))))
